db/migrate/20160401043112_create_phones.rb

```diff
@@ -5,7 +5,7 @@ def change
 
       t.string :number, null: false
       t.string :extension
-      t.string :type, null: false
+      t.string :service_type, null: false
```

Comment:
It's usually bad practice to edit migrations for the reason you mention: devs will have to recreate their local DBs. Another migration that renames the column is a better way to go. There's a `rename_column` method for migrations that makes this pretty easy.